src/SlashCommands.tsx

```diff
@@ -970,6 +971,51 @@ export const Commands = [
         },
         category: CommandCategories.advanced,
     }),
+    new Command({
+        command: "query",
+        description: _td("Opens chat with the given user"),
+        args: "<user-id>",
+        runFn: function(roomId, userId) {
+            if (!userId || !userId.startsWith("@") || !userId.includes(":")) {
+                return reject(this.getUsage());
+            }
+
+            return success((async () => {
+                dis.dispatch({
+                    action: 'view_room',
+                    room_id: await ensureDMExists(MatrixClientPeg.get(), userId),
+                });
+            })());
+        },
+        category: CommandCategories.actions,
+    }),
+    new Command({
+        command: "msg",
+        description: _td("Sends a message to the given user"),
+        args: "<user-id> <message>",
+        runFn: function(_, args) {
+            if (args) {
+                const matches = args.match(/^(\S+?)(?: +(.*))?$/s);
```

Comment:
A comment to explain what we're looking for would be good. 

https://xkcd.com/1171/ etc